### ISSUE TITLE
No such file or directory: 'output/Conv_bn_moving_mean.dat' 

### ISSUE THREAD

**Issue by biycyr** [chuanqi305/MobileNetv2-SSDLite#38]
I have this problem while using load_caffe_weights.py.
I have checked ssdlite_mobilenet_v2_coco_2018_05_09/frozen_inference_graph.pb and there is no tensor with 'Conv' and 'moving_mean' in the name. Does any one have similar problem?

**Comment by biycyr** [comment]
It seems that tensorflow model has merge batchnorm  of _Conv_ with convolution of _Conv_. **FeatureExtractor/MobilenetV2/Conv/BatchNorm/batchnorm/mul_1** and **FeatureExtractor/MobilenetV2/Conv/BatchNorm/batchnorm/add_1** are now Conv2D op and addtion op. Does this mean the load_caffe_weights.py needs a rewrite? @chuanqi305

**Comment by woshilicong33** [comment]
How to solve the problem?

**Comment by biycyr** [comment]
I didn't. I choose to use Mobilenetv2_SSD(not ssdlite). In which, batchnorm layers are not merged yet. Although you may still need to change the way of naming .dat in  dump_tensorflow_weights.py, at least it is easier to change the network structure.

**Comment by biycyr** [comment]
My fault. Just add --nobn true while using gen_model.py
